### ISSUE TITLE
Errors when using v1.10.0 on a dedicated server (1.20.2)

### ISSUE THREAD

**Issue by IonicEcko** [Steveplays28/realisticsleep#55]
Server: FabricMC 1.20.2 (0.15.1)
Mods: 
- cloth-config-12.0.111-fabric
- fabric-api-0.91.2+1.20.2
- Server-Translations-2.1.0+1.20.2-rc2
- realisticsleep-1.10.0+mc1.20.2

What is happening:
Server won't launch, crashes during launch with the following error
```
09/12/2023 [20:48:19] [main/ERROR]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'realisticsleep'!
    at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:63) ~[fabric-loader-0.15.1.jar:?]
    at net.minecraft.server.Main.main(Main.java:109) ~[server-intermediary.jar:?]
    at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) ~[fabric-loader-0.15.1.jar:?]
Caused by: java.lang.RuntimeException: Cannot load class net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents in environment type SERVER
    at net.fabricmc.loader.impl.transformer.FabricTransformer.transform(FabricTransformer.java:59) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPreMixinClassByteArray(KnotClassDelegate.java:462) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:415) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.15.1.jar:?]
    at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.15.1.jar:?]
    at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
    at com.github.steveplays28.realisticsleep.RealisticSleep.onInitialize(RealisticSleep.java:47) ~[realisticsleep-1.10.0+mc1.20.2.jar:?]
    at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384) ~[fabric-loader-0.15.1.jar:?]
    ... 6 more
Process exited with code: [0]  termSignal: []09/12/2023 Starting net.fabricmc.loader.impl.game.minecraft.BundlerClassPathCapture
```

Reverting to 1.9.2 fixes

**Comment by Steveplays28** [comment]
Hi IonicEcko, thanks for the report.
I'll look into this issue.

**Comment by Steveplays28** [comment]
This issue has been fixed in [`v1.10.1`](https://github.com/Steveplays28/realisticsleep/releases)!
